Council/xgov_council-18.md

```diff
@@ -0,0 +1,27 @@
+---
+id: 18
+author: Taras Hirniak (@Tarpoz)
+email: taras.trombon@gmail.com
+address: 67OLU3IO4YWOAYJK273DQJS6RU2ES45WPAYHWNIQJNYKRXGZ5NPBMMEK3I
+status: Draft
```

Comment:
When you are ready just change the status from Draft to Final and I will merge the PR!